dashboard/app/models/pd/application/facilitator1920_application.rb

```diff
@@ -148,138 +148,51 @@ def self.filtered_labels(course)
       FILTERED_LABELS[course]
     end
 
-    # @override
-    def self.csv_header(course, user)
-      # strip all markdown formatting out of the labels
-      markdown = Redcarpet::Markdown.new(Redcarpet::Render::StripDown)
-      CSV.generate do |csv|
-        columns = filtered_labels(course).values.map {|l| markdown.render(l)}.map(&:strip)
-        columns.push(
-          'Status',
-          'Locked',
-          'General Notes',
-          'Notes 2',
-          'Notes 3',
-          'Notes 4',
-          'Notes 5',
-          'Question 1 Support Teachers',
-          'Question 2 Student Access',
-          'Question 3 Receive Feedback',
-          'Question 4 Give Feedback',
-          'Question 5 Redirect Conversation',
-          'Question 6 Time Commitment',
-          'Question 7 Regional Needs',
-          'Regional Partner'
-        )
-        csv << columns
+    # Filter out extraneous answers based on selected program (course)
```

Comment:
This function creates a list of columns to filter, correct? If so, can you update this column to a bit clearer?